### PR TITLE
Add users and closets

### PR DESCRIPTION
**Pull request by ahmedjahmi**

### CODE REVIEW

File: 0001_initial_schema.up.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  location_state CHAR(100),
+  is_curator BOOLEAN,
+  email CHAR(100),
+  instagram CHAR(100),
+  facebook CHAR(100),
+  tiktok CHAR(100),
+  whatsapp CHAR(100),
+  twitter CHAR(100),
+  pinterest CHAR(100),
+  last_login TIMESTAMP,
```
Comment (jmuru) [inline 0001_initial_schema.up.sql:19]: should any of these fields be not nullable?

**Comment by jmuru** [comment]
You also need to add the rollback tables to the .down.sql files

File: 0001_initial_schema.up.sql (hunk 1 above)
Comment (ahmedjahmi) [reply 0001_initial_schema.up.sql:19]: username and email stand out to me as eligible for not nullable.

Comment (ahmedjahmi) [reply 0001_initial_schema.up.sql:19]: also will add a last_name field

Comment (jmuru) [reply 0001_initial_schema.up.sql:19]: lets go with that!

**Review by jmuru — APPROVED** [review]

**Pull request merged**